js/utils.js

```diff
@@ -0,0 +1,27 @@
+const utils = {
```

Comment:
write please:
`export const utils`
and 
`export function checkInitialTodoList()`
instead of this one 
`export { utils, checkInitialTodoList };`